### PR TITLE
Allow to configure max polling interval via host.json

### PR DESCRIPTION
**Pull request by myarotskaya**
Makes the maximum queue polling interval configurable via host.json.

Second step to resolving [Azure/azure-functions-durable-extension#618](https://github.com/Azure/azure-functions-durable-extension/issues/618).

### CODE REVIEW

File: src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs [hunk 1]
```diff
@@ -210,6 +210,12 @@ public class DurableTaskOptions
         /// <value>Assembly qualified class name that implements <see cref="ILifeCycleNotificationHelper">ILifeCycleNotificationHelper</see>.</value>
         public string CustomLifeCycleNotificationHelperType { get; set; }
 
+        /// <summary>
+        /// Gets or sets the maximum queue polling interval.
+        /// </summary>
+        /// <value>Maximum interval for polling control and work-item queues.</value>
+        public TimeSpan MaxQueuePollingInterval { get; set; } = TimeSpan.FromSeconds(30);
```
Comment (myarotskaya) [inline src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs]: @cgillum I think it should come from here https://github.com/Azure/durabletask/blob/azure-storage/src/DurableTask.AzureStorage/AzureStorageOrchestrationServiceSettings.cs#L28
What do you think?

Comment (cgillum) [reply src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs]: I think it's fine to keep this the way it is.

Comment (cgillum) [inline src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs]: Let's also add this property value to the `GetDebugString()` method further below. This is important for diagnostics,

Comment (myarotskaya) [reply src/WebJobs.Extensions.DurableTask/DurableTaskOptions.cs]: Done!

**Comment by myarotskaya** [comment]
@cgillum I noticed that `DurableTaskEndToEndTests.ActorOrchestration` test is failing when `extendedSessions` parameter is true. It looks like the reason is in the new version of `Microsoft.Azure.DurableTask.AzureStorage` package (Job host started to generate log messages with `IsReplay: True` while according to test it shouldn't). What would you suggest to do?

**Comment by cgillum** [comment]
Yes I noticed this as well. It's a bug in the latest version of DurableTask.Core that I'm currently investigating. My hope is to resolve this today and re-run the CI so that your PR becomes green.

**Comment by jeffhollan** [comment]
@myarotskaya this is awesome! Thanks a ton for contributing 🥳

**Comment by myarotskaya** [comment]
@cgillum @jeffhollan thanks guys! I'm so glad that we figured it out so quickly!

**Pull request merged**